bindata/network/dhcp/dhcp-daemon.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: dhcp-daemon
```

Comment:
could this just be added to the existing multus DaemonSet as an optional additional container? (using `{{if ...}}` to optionally include/exclude part of the template based on a variable set by the renderer, as in `bindata/network/openshift-sdn/004-multitenant.yaml`)